### ISSUE TITLE
Shouldn't this use XDG_CONFIG_HOME?

### ISSUE THREAD

**Issue by challsted** [KenjiTakahashi/td#4]
As the title suggests, shouldnt this be storeing everything under `$XDG_CONFIG_HOME/td/<files here>` instead of `~/<files here>`?

Not sure how hard it would be to change the code to do this.

**Comment by KenjiTakahashi** [comment]
Yeah, if I was writing it today I would have followed the XDG spec. Back then I did not :-). It should not be hard to implement, but since it's already out there in current state, backwards compatibility should be considered as well.
There are a few things that I'd like to improve in td, so maybe I'll do this as well at some point, but it's not high priority to me, sorry.
